Vedtekter for Spillmakerlauget Norge.md

```diff
@@ -57,16 +64,44 @@ Landsmøtet skal behandle følgende saker:
 1.	Vedta budsjett.
 1.	Behandle innkomne forslag og saker.
 1.	Velge:
-    - Leder og eventuelt nestleder annethvert år
-    - Revisor (hvis foreningen er revisjonspliktig)
+    - Leder og nestleder
+    - Revisor, hvis foreningen er revisjonspliktig
 
-Leder og nestleder velges enkeltvis.
+Leder og nestleder velges enkeltvis. De øvrige medlemmer til styret velges samlet. Deretter velges eventuelle varamedlemmer samlet. Rekkefølgen kan ved behov avgjøres ved skriftlig valg i forhold til stemmetall. Alle styremedlemmer blir valgt for ett år om gangen.
 
-## § 9 Ekstraodinære landsmøter
-Det avholdes ekstraordinært landsmøte hvis over en tredjedel av medlemsorganisasjonene eller over halvparten av styret i Spillmakerlauget Norge krever det. Innkalling må sendes ut med minst 2 ukers varsel og kun sakene nevnt i innkallingen kan tas opp.
+## § 9 Ekstraordinære landsmøter
+Ekstraordinære landsmøter holdes når styret bestemmer det, eller minst en tredjedel av de stemmeberettigede medlemmene krever det.
+Det innkalles på samme måte som for ordinære landsmøter, med minst 1 ukes varsel.
```

Comment:
Countersuggestion to keep it at 2 weeks, if we are concerned that min 1 week could be too soon for people to plan around and participate?

Reply:
Noted. If a majority wants this, then I will add a commit where "1 ukes" is replaced by " 2 ukers" before merging to main branch.

Reply:
2 weeks was decided as deadline for inviting to extra-ordinary general assemblies.